src/components/organisms/UiSidePanel/UiSidePanel.vue

```diff
@@ -355,6 +355,10 @@ onBeforeUnmount(() => {
   z-index: 1;
 
   &__dialog {
+    @include mixins.use-logical($element, padding, 0);
```

Comment:
FYI: keep
```
  padding: 0;
  border-width: 0;
  margin: 0;
```
this style reset dialog default styles, IMO we should keep it without changes